Kentico.Kontent.Urls/Delivery/DeliveryEndpointUrlBuilder.cs

```diff
@@ -51,6 +51,7 @@ public DeliveryEndpointUrlBuilder(IOptionsMonitor<DeliveryOptions> deliveryOptio
         /// Initializes the URL builder using <see cref="IOptionsMonitor{TOptions}"/>. Ideal for simple scenarios where hot reloading of configuration is not required.
         /// </summary>
         /// <param name="deliveryOptions">The configuration object.</param>
+        [Obsolete]
```

Comment:
I think it might be worth it to edit the documentation of the constructor as well

Reply:
Okay, I will at enhance this attribute with description parameter (to use `IOptionsMontor` version instead)